client/components/Reports/Reports.jsx

```diff
@@ -23,24 +28,36 @@ const Reports = () => {
 
   if (loading) {
     return (
-      <PageStatus
-        title="Loading - AT Interop Reports | ARIA-AT"
-        heading="Assistive Technology Interoperability Reports"
-      />
+      <Container
+        id="main"
+        as="main"
+        tabIndex="-1"
+        className={clsx(
+          commonStyles.fhContainer,
+          reportsStyles.reportsContainer
+        )}
+      >
+        <Helmet>
+          <title>Loading - AT Interop Reports | ARIA-AT</title>
+        </Helmet>
+        <KeyMetricsBanner />
+        <section className={clsx(reportsStyles.contentSection)}>
+          <h1>Assistive Technology Interoperability Reports</h1>
+          <p>Loading...</p>
+        </section>
+      </Container>
     );
```

Comment:
I separated the `KeyMetricsBanner` from report summary to allow the same "slightly wider than the body" look of the home page